capistrano3-puma.gemspec

```diff
@@ -19,5 +19,4 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
 
   spec.add_dependency 'capistrano', '~> 3.0'
-  spec.add_dependency 'puma' , '>= 2.6'
```

Comment:
Ok, sure. But why would you need it? I guess for constraining puma version, but, unfortunately, this will not work if your capistrano configs and the repo that is being deployed have separate Gemfiles.